### PR TITLE
[ricos-plugins] external formatting toolbar support toolbar settings

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/ricos-plugins/web/src/plugin-text-button.ts [hunk 1]
```diff
@@ -204,18 +211,26 @@ export class PluginTextButtons implements FormattingToolbarButtons {
   }
 
   toExternalToolbarButtonsConfigs(
-    editorCommands: EditorCommands
+    editorCommands: EditorCommands,
+    isMobile: boolean
```
Comment (maimonav) [inline packages/ricos-plugins/web/src/plugin-text-button.ts:215]: No need to pass `isMobile` prop, you can access it via `this.services.context.isMobile`

File: packages/ricos-types/web/src/formatting-toolbar-buttons.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ export interface FormattingToolbarButtons {
   ) => IToolbarItemConfigTiptap[];
 
   toExternalToolbarButtonsConfigs(
-    editorCommands: EditorCommands
+    editorCommands: EditorCommands,
+    isMobile: boolean
```
Comment (maimonav) [inline packages/ricos-types/web/src/formatting-toolbar-buttons.ts:41]: Please revert

File: packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx [hunk 3]
```diff
@@ -71,7 +71,10 @@ export class RichContentAdapter implements TiptapAdapter {
           ? services.plugins.getAddButtons()
           : services.plugins.getTextButtons();
       return {
-        buttons: buttons.toExternalToolbarButtonsConfigs(this.getEditorCommands()),
+        buttons: buttons.toExternalToolbarButtonsConfigs(
+          this.getEditorCommands(),
+          services.context.isMobile
```
Comment (maimonav) [inline packages/tiptap-editor/web/src/components/RichContentAdapter/RichContentAdapter.tsx:76]: Please revert

**Review by maimonav — CHANGES_REQUESTED** [review]
Please follow the comments :)

File: packages/ricos-plugins/web/src/plugin-text-button.ts (hunk 1 above)
Comment (noamtwix) [reply packages/ricos-plugins/web/src/plugin-text-button.ts:216]: `services` is an entity of `PluginTextButton` and I need it in `PluginTextButtons`

**Pull request merged**